xml/src/main/java/org/opennms/integration/api/xml/schema/eventconf/Event.java

```diff
@@ -61,6 +61,12 @@
     @XmlElement(name="uei", required=true)
     private String m_uei;
 
+    /**
+     * The Priority of the Event definition. Higher number has higher priority.
+     */
+    @XmlElement(name = "priority", required = false)
+    private Integer m_priority;
```

Comment:
missing `m_priority` in equals() and hashCode()

Reply:
yup. thanks for both.